Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml

```diff
@@ -1,5 +1,7 @@
 id: Expanse Load-Create List
 version: -1
+contentitemexportablefields:
+  contentitemfields: {}
 name: Expanse Load-Create List
 description: |
   Sub-playbook to support Expanse Handle Incident playbook.
```

Comment:
Maybe change the description to be more specific?

Reply:
It is a multiple line description, maybe you missed the lower lines?